importer/tag/src/test/java/org/itsallcode/openfasttrace/importer/tag/TestTagImporter.java

```diff
@@ -268,6 +268,29 @@ void tagWithMultipleRequiredCoverageWithSpaces()
                 itemWithReadableName(COVERING_ARTIFACT_TYPE1, 1, ID1, List.of(NEEDED_COVERAGE1, NEEDED_COVERAGE2)));
     }
 
+    // [utest->dsn~import.full-coverage-tag-with-revision~1]
+    @Test
+    void tagWithRevision()
+    {
+        assertItems(
+                tag("impl~~42", "req~example_name~17"), //
```

Comment:
```suggestion
                tag("impl~~42", "req~example_name~17"), //
```

While this is compact, it hides the actual test. I recommend writing out the complete tags text. This way the setup is much more explicit. Also, It is about the same number of characters to type. :-)

Reply:
I had this before, but it caused problems when tracing OFT's own requirements. The tag for testing was recognized as a real tag.